sos/report/plugins/ovn_central.py

```diff
@@ -76,6 +76,15 @@ def setup(self):
             if ovs_rundir:
                 self.add_copy_spec(os.path.join(ovs_rundir, pidfile))
 
+        if self.get_option("all_logs"):
+            self.add_copy_spec([
+                "/var/log/ovn/",
+            ])
+        else:
+            self.add_copy_spec([
+                "/var/log/ovn/*.log",
+            ])
```

Comment:
For singleton collections, `add_copy_spec()` can take a string instead of a list.